packages/common/src/interaction.ts

```diff
@@ -164,10 +165,8 @@ export interface InteractionForkPayload {
  * @see async oparameter of InteractionExecutionPayload
  */
 export interface ExecuteInteractionWorkflowParams {
-    interactionName: string, // usefull for logging
-    runId: string,
-    requestorAccountId: string,
-    requestorProjectId: string,
+    interaction_name: string, // usefull for logging
```

Comment:
This changes from camel case to snake case. Is there any compatibility concerns on the workflows or APIs?